src/m_menu.c

```diff
@@ -3981,7 +3988,18 @@ setup_menu_t gen_settings3[] = { // General Settings screen3
 
   {"", S_SKIP, m_null, M_X, M_Y + general_end5*M_SPC},
 
-  {"Miscellaneous"  ,S_SKIP|S_TITLE, m_null, M_X, M_Y + general_title6*M_SPC},
```

Comment:
Wow, this sounds almost frightening. 😁

Reply:
Maybe you could drop the "significant" part?

Reply:
"Core" game mechanics?

Reply:
How about "Incompatible Game Mechanics Changes"?

Reply:
Pistol start isn't really about "mechanics", so I think "Incompatible Game Changes" could fit. @kraflab @mikeday0 I think we could use some native speaker support here, please.

Reply:
Hmm. A standalone "Incompatible" could be a little confusing. Maybe something like "Compatibility-breaking Features"? Or maybe "Demo-unsafe Features" would get the point across.

Reply:
> "Demo-unsafe Features"

Would like to vote for this. Many users knows for sure what does "Compatibility" standard means, but some are probably not, it is impossible speak for everyone. "Demo" prefix gives more direct point IMO, because yeah, for single player game it is possible to do, or fix, or _break_ whatever you want, but keeping demo sync is ... well, you know what is that means. 🙃

Reply:
But these features can literally break compatibility to the point where the map can't be completed. I haven't found it yet, but I'm sure it exists. 😆 I vote for "Compatibility-breaking Features".

Reply:
> "Compatibility-breaking Features".

Yep, that's it!

Reply:
Aaaah, true, I even left a [comment](https://www.doomworld.com/forum/topic/130846-what-source-ports-fix-the-buggy-blockmap/?do=findComment&comment=2533037) on DoomWorld about map breaking - nuts.wad can't be finished normal way for sure because of blockmap fix.